docs/source/java/jdbc.rst

```diff
@@ -172,3 +172,101 @@ The JDBC to Arrow type mapping can be obtained at runtime from
++----------------------------+----------------------------+-------+
+| Decimal128                 | DECIMAL (setBigDecimal)    |       |
++----------------------------+----------------------------+-------+
+| Decimal256                 | DECIMAL (setBigDecimal)    |       |
++----------------------------+----------------------------+-------+
+| FixedSizeBinary            | BINARY (setBytes)          |       |
++----------------------------+----------------------------+-------+
+| Float32                    | REAL (setFloat)            |       |
++----------------------------+----------------------------+-------+
+| Int8                       | TINYINT (setByte)          |       |
++----------------------------+----------------------------+-------+
+| Int16                      | SMALLINT (setShort)        |       |
++----------------------------+----------------------------+-------+
+| Int32                      | INTEGER (setInt)           |       |
++----------------------------+----------------------------+-------+
+| Int64                      | BIGINT (setLong)           |       |
++----------------------------+----------------------------+-------+
+| LargeBinary                | LONGVARBINARY (setBytes)   |       |
++----------------------------+----------------------------+-------+
+| LargeUtf8                  | LONGVARCHAR (setString)    | \(1)  |
++----------------------------+----------------------------+-------+
+| Time[s]                    | TIME (setTime)             |       |
++----------------------------+----------------------------+-------+
+| Time[ms]                   | TIME (setTime)             |       |
++----------------------------+----------------------------+-------+
+| Time[us]                   | TIME (setTime)             |       |
++----------------------------+----------------------------+-------+
+| Time[ns]                   | TIME (setTime)             |       |
++----------------------------+----------------------------+-------+
+| Timestamp[s]               | TIMESTAMP (setTimestamp)   | \(2)  |
++----------------------------+----------------------------+-------+
+| Timestamp[ms]              | TIMESTAMP (setTimestamp)   | \(2)  |
++----------------------------+----------------------------+-------+
+| Timestamp[us]              | TIMESTAMP (setTimestamp)   | \(2)  |
++----------------------------+----------------------------+-------+
+| Timestamp[ns]              | TIMESTAMP (setTimestamp)   | \(2)  |
++----------------------------+----------------------------+-------+
+| Utf8                       | VARCHAR (setString)        |       |
++----------------------------+----------------------------+-------+
+
+* \(1) Strings longer than Integer.MAX_VALUE bytes (the maximum length
+  of a Java ``byte[]``) will cause a runtime exception.
+* \(2) If the timestamp has a timezone, the JDBC type defaults to
+  TIMESTAMP_WITH_TIMEZONE.  If the timestamp has no timezone,
```

Comment:
what would happen when a timezone is absent, the program would thrown an exception?

Reply:
It'll just call `setTimestamp(int, Timestamp)` instead of `setTimestamp(int, Timestamp, Calendar)`, I'll update the doc

Reply:
Thanks for the clarification.